mers/NEWS.md

```diff
@@ -31,11 +76,23 @@ This release enables execution on Intel® Processor Graphics.
   - MSDK ->  `intel-mediasdk-19.4.0`
   - FFMPEG ->  `n4.2`
   - FFMPEG_MA_RELEASE ->  `0.4`
-  - GST ->  `1.16.0`
+  - GStreamer* (GST)
+    * GST PLUGINS BASE -> `1.16.0`
+    * GST PLUGINS GOOD -> `1.16.0`
+    * GST PLUGINS BAD -> `1.16.0`
+    * GST PLUGINS UGLY -> `1.16.0`
   - OPENCV ->  `4.1.2`
   - DLDT ->  `2019_R3.1`
   - VA_GSTREAMER_PLUGINS ->  `0.6.1`
 
+- Intel highlights:
+  - Enable Intel® IPP (ICV version) 2019.0.0 on OpenCV
+  - Enable Intel® Math Kernel Library for Deep Neural Networks (Intel® MKL-DNN) `v1.0.1` on DLDT
```

Comment:
This changed names to onednn, please add the correct format

Reply:
You are ryt.

Reply:
Done